inst/WORDLIST

```diff
@@ -0,0 +1,2 @@
+BMI
```

Comment:
List of words incorrectly flagged by spell check so they can be ignored by the check. These words will also be treated as exceptions by CRAN when they run their own spell check